signin/style.css

```diff
@@ -0,0 +1,146 @@
+* {
```

Comment:
I've not seen any media queries to support other screen sizes. Or does this not need it?